src/decision/dMCDA.jl

```diff
@@ -447,8 +447,8 @@ function create_shade_matrix(
         wt_hi_cover,
     ]
 
-    SH[:, 4] = (1.0 .- A[:, 4]) # complimentary of wave damage risk
-    SH[:, 8] = A[:, 8] # total area of coral cover
+    SH[:, 4] = (1.0 .- SH[:, 4]) # complimentary of wave damage risk
+    SH[:, 8] = SH[:, 8] # total area of coral cover
```

Comment:
`SH[:, 8] = SH[:, 8] # total area of coral cover`

Want to think about this one for a little bit? 😝

Reply:
two spaces after code for inline comments please

Reply:
whoops didn't even see this :D